render.js

```diff
@@ -0,0 +1,16 @@
+var render = (function() {
+
+  var MovieInfo = function(response) {
+
+  }
+
+  var MovieGIF = function(response) {
```

Comment:
The function name "MovieGIF" should start with a lower case character. Try changing it to "movieGIF"